kvirt/k3s/__init__.py

```diff
@@ -49,6 +49,7 @@ def create(config, plandir, cluster, overrides):
     data['kube'] = data['cluster']
     masters = data.get('masters', 1)
     network = data.get('network', 'default')
+    sdn = data.get('sdn', 'flannel')
```

Comment:
I would love to get the extra arg `sdn` out at some point. Now that - with this implementation - all K3s install args and K3s executable args are respected if defined.
As one can therefore just define the needed args to avoid *flannel* being installed. I know we had a word or two on Slack about it. Where you said that no one is forced to use it. Which is 100% correct. However, I think it makes KCLI easier to use if it's not there - as it makes things cleaner in regards to deploying K3s with *KCLI*.

Reply:
well i try to map what i have in other cluster workflows.
for openshift or kubeadm, the user can specify target sdn within the supported ones and not having to worry about implementation details made to make it available on the deployment

Reply:
Okay that makes sense. Good argument for having it. I just though I was the one asking for it and that was why it was implemented for K3s at least. I wasn't aware of the same option for *kubeadm* and *openshift*.

Would it then be possible to set the `sdn` parameter to a value that makes it so, that flannel is not installed and nothing else happens. No bootstrap of no CNI at all?

Reply:
yes, that makes sense

Reply:
I would love that then ... writing from my company account.

Reply:
addressed this by making sure that if sdn is unset (that is `sdn:` in a parameter file), flannel gets disabled and no further action is taken

Reply:
changed the code to remove curl at the end of the execution of each node